tool/full-test.sh

```diff
@@ -429,7 +429,7 @@ step "Building the Agda base library"        \
      "Built Agda base library successfully"  \
      "Failed to build the Agda base library" \
      "Canceled Agda base library build"      \
-     "./tool/check-agda.sh --recompile base/agda"
+     "./tool/check-agda.sh base/agda"
```

Comment:
Why did you remove the `--recompile` flag?

Reply:
Compiling the Agda base library takes long time (compared to the other tests in the script). Furthermore Agda notices if files change and the `.agdai` files become invalid. I think this change saves most people working with the compiler some time and should not hurt the test itself.